src/web_interface/components/miscellaneous_routes.py

```diff
@@ -37,6 +39,12 @@ def _app_home(self):
         stats.shutdown()
         return render_template('home.html', general_stats=general_stats, latest_firmware_submissions=latest_firmware_submissions,
                                latest_comments=latest_comments, latest_comparison_results=latest_comparison_results)
+    
+    def _app_plugins(self):
```

Comment:
You need to add a `@roles_accepted` decorator to the function (see lines 53, 67).
I would suggest using the privilege **view_analysis**.

This also causes a test in *src/test/acceptance/test_authenticated_gui.py* to fail.